### DISCUSSION TITLE
v0.13.1

### DISCUSSION THREAD

**Discussion by ZachNagengast** [argmaxinc/WhisperKit#344] [Announcements]
Patch release to fix some issues reported relating to tokenizer loading and config-based logit filters.
- Tokenizer downloading now respects downloadBase if specified https://github.com/argmaxinc/WhisperKit/issues/339 thanks for the suggestion @Kavi-Gupta
- Tokenizer will now load offline with the CLI if it exists in the modelFolder path https://github.com/argmaxinc/WhisperKit/issues/340 thanks for reporting @cedricporter
- Logits filters were observed never actually being passed to the text decoder if defined in the config, this patch makes sure they are observed by passing them to the text decoder on WhisperKit initialization.

Also includes improved logging contributed by @JimLiu, thanks everyone for helping make WhisperKit ever better! 🚀 

## What's Changed
* feat: enhance verbose logging in WhisperKit CLI by @JimLiu in https://github.com/argmaxinc/WhisperKit/pull/335
* Pass logitfilters to textdecoder, improve tokenizer loading by @ZachNagengast in https://github.com/argmaxinc/WhisperKit/pull/343

**Full Changelog**: https://github.com/argmaxinc/WhisperKit/compare/v0.13.0...v0.13.1

<hr /><em>This discussion was created from the release <a href='https://github.com/argmaxinc/WhisperKit/releases/tag/v0.13.1'>v0.13.1</a>.</em>
